client/gutenberg/extensions/publicize/connection.jsx

```diff
@@ -81,25 +120,31 @@ class PublicizeConnection extends Component {
 			/>
 		);
 
-		if ( disabled || this.connectionIsFailing() ) {
+		if ( disabled || this.connectionIsFailing() || this.connectionNeedsReauth() ) {
 			toggle = <Disabled>{ toggle }</Disabled>;
 		}
 
 		return (
 			<li>
+				{ this.maybeDisplayGooglePlusNotice( serviceName ) }
```

Comment:
I know this wasn't introduced here, but it feels weird -- we apparently know the service name because this method is dedicated to `google-plus` - why do we need to pass it as a parameter anyway?

Reply:
Makes sense but won't address it here. Can you make an issue and assign me? Thanks.

Reply:
https://github.com/Automattic/wp-calypso/issues/30787